### ISSUE TITLE
 Error with Nanostring CosMx Subcellular Lung Cancer example: Error in calculateOverlapRaster

### ISSUE THREAD

**Issue by khumarahn** [giotto-suite/Giotto#564]
Hi! Trying to literally follow the Nanostring CosMx Subcellular Lung Cancer example. It fails in Section 6 with
```
Error in calculateOverlapRaster(fov_join, feat_info = "neg_probe") : 
  trying to get slot "spatVector" from an object of a basic class ("NULL") with no slots
Execution halted
```
Here is the complete output: [test.txt](https://github.com/drieslab/Giotto/files/10788330/test.txt)

Using a fresh installation of giotto in R-4.2.1

**Comment by RubD** [comment]
Hi @khumarahn can you try it again with our `suite_dev` branch? This should be solved.

**Comment by mattobny** [comment]
@khumarahn This should be fixed. The tutorial has been fixed as of the version bump to v3.3. Please open a new issue if necessary and we will gladly assist you!
